xbmc/interfaces/legacy/ModuleXbmcplugin.h

```diff
@@ -103,9 +103,53 @@ namespace XBMCAddon
      *                           - SORT_METHOD_LABEL_IGNORE_THE, SORT_METHOD_VIDEO_SORT_TITLE,
      *                           - SORT_METHOD_VIDEO_SORT_TITLE_IGNORE_THE, SORT_METHOD_FULLPATH,
      *                           - SORT_METHOD_LABEL_IGNORE_FOLDERS, SORT_METHOD_CHANNEL
-     * 
+     *   - sortMethods available:
```

Comment:
imo python devs should not start digging in core code to know what they can use.